src/pages/dataElements/fields/AggregationTypeField.tsx

```diff
@@ -18,18 +31,7 @@ const aggregationTypeDisabledHelpText = i18n.t(
 export function AggregationTypeField() {
     const { change } = useForm()
     const { values } = useFormState({ subscription: { values: true } })
```

Comment:
Isn't it better to use `useField('valueType`) instead of subscribing to all the values here?